docs/docsite/rst/contributor/communication.rst

```diff
@@ -0,0 +1,39 @@
+.. _communication:
+
+Communication
+=============
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+.. _code_of_conduct:
+
+Code of Conduct
+---------------
+
+All communication and interactions in the Ansible Community are governed by our :ref:`code_of_conduct`. Please read and understand it!
```

Comment:
What about `s/understand it/abide by it`?

Saying "understand it" comes across slightly like talking down to the reader. It's not difficult to understand the COC. And we really want folks to act according to the code rather than just understanding it.

Reply:
SGTM, please suggest the change and commit it